lib/account/account-service.js

```diff
@@ -1,70 +1,74 @@
 "use strict";
 
 /**
- * What we initially send back to the client so they can start to initiate
- * the complete procedure and save information.
- *
- * @typedef {Object} AccountService~initiateResult
- * @property {string} accountId
- * @property {string} email
- * @property {string} mfaKey
- * @property {string} salt
+ * Handles the creating, deletion, and updating of account and
+ * registration files.
  */
 
 /**
- * Stored information for the account.  This is changed from the initiated
- * file as we add password and there could be other items added after
- * we initiate the account.
+ * Information about an account holder. Used to generate/validate
+ * a password and two factor authentication as well as Ids and
```

Comment:
Ids is not a proper noun and thus should be lowercase or spelled out as "identifiers"